Microsoft.Toolkit.Uwp.UI.Controls.Core/TextToolbar/TextToolbar.Events.cs

```diff
@@ -295,6 +309,14 @@ public bool ControlKeyDown
             get { return IsKeyActive(CoreWindow.GetForCurrentThread().GetKeyState(VirtualKey.Control)); }
         }
 
+        /// <summary>
+        /// Gets a value indicating whether Menu is pressed down
+        /// </summary>
+        public bool MenuKeyDown
```

Comment:
I don't know if that's necessary to expose this property, I just followed what was done for the other keys.

Reply:
Yeah, not sure why it was done this way initially, but we can keep it the same for now. 🙂